order online.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/order online.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Questrial&family=Raleway&family=Roboto:wght@300&display=swap"
+        rel="stylesheet">
+    <title>Responsive grid project order online page</title>
+</head>
+<body>
+    <header>
+        <button class="button-color-hover"><a href="store/html">Order Online</a></button>
+        <div class="header-user-icon"><a href="#"><img src="images/user-circle-solid.svg" alt="icon"/>Log In</a></div>
+        <div class="header-logo"><a href="header">Juice Bar</a></div>
+        <div class="header-hamburger-icon"><a href="#"><img src="images/grip-lines-solid.svg" alt="icon"/></a></div>
+    </header>
+
+
+    <main>
+        <section class="menu-section">
+        <div class="user-icon"></div>
+        <p>Hi, Guest</p>
+        <ul>
+            <li>Menu</li>
+            <li>Cold Pressed J...</li>
+            <li>Smoothies</li>
+            <li>Wellness Shots</li>
+            <li>Cleanse Kit</li>
+        </ul>
+        </section>
+
+        <h1>Menu</h1>
+        <section class="cold-pressed">
+            <h2 class="product-header">Cold Pressed Juice</h2>
+            <div class="up-icon"></div>
+
+            <div class="cold-pressed-img1"></div>
+            <h3 class="product-name">Tangerine</h3>
+            <p class="product-price">$8</p>
+            <p class="product-description">I’m a dish description. Click “Edit Menu” to open the ...<div class="full-text">Restaurant Menu editor and change my text.</div></p>
```

Comment:
Try to get in the habit of wrapping elements that 'go together' into a div so that they can more easily be styled, moved around, and aligned.